.golangci.yml

```diff
@@ -18,6 +22,22 @@ linters-settings:
       - prefix(github.com/aws/eks-anywhere)
     custom-order: true
     skip-generated: false
+  godot:
+    exclude:
+      # Exclude todo comments.
+      - "(?i)^\\s*todo:"
```

Comment:
We're still in draft sooo, perhaps drop the `:`. I usually add a BOM to my todo comments so I can `grep` easily in the future.

e.g.

`TODO(chrisdoherty)`